fastlane/metadata/en-CA/name.txt

```diff
@@ -1 +0,0 @@
-WordPress – Website Builder
```

Comment:
Another case of `429 Too Many Requests` on `gp_downloadmetadata` I guess… — which sadly, as mentioned in some P2 comments, [does swallow any error under `UI.message "No translation available for #{target_locale}"` regardless of the response's HTTP error code], making it so easy to miss… 😞